packages/functional-tests/tests/oauth/syncSignIn.spec.ts

```diff
@@ -42,7 +50,18 @@ test.describe('severity-1 #smoke', () => {
       await relier.goto();
       await relier.clickEmailFirst();
       await login.useDifferentAccountLink();
-      await login.fillOutFirstSignUp(email, PASSWORD);
+
+      // This conditional is temporary until we have a test explicitely for this in React.
+      // When that happens, we can skip this entire test if the flag is on instead. For
+      // now, it's compatible with `fullProdRollout: true` _and_ `fullProdRollout: false`.
+      // eslint-disable-next-line playwright/no-conditional-in-test
+      if (config.showReactApp.signUpRoutes !== true) {
+        await login.fillOutFirstSignUp(email, PASSWORD);
+      } else {
+        await signupReact.fillOutEmailForm(email);
+        await signupReact.fillOutSignupForm(PASSWORD, '21');
+        await signupReact.fillOutCodeForm(email);
```

Comment:
We could probably combine these into a signupReact.fillOutFirstSignUp(email, PASSWORD, '21')